app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    list_commands = command.split()
```

Comment:
```suggestion
    commands = command.split()
```
list_ prefix is redundant